README.md

```diff
@@ -32,22 +31,13 @@ audited, might contain bugs and should not be used in production.
 
 ## Introduction
 
-For Aave, one of the main current and future goals is growth of liquidity and
-user base. As seen on side-chains with low transaction cost like Polygon or
-Avalanche, there is high demand to use the Aave protocol with small amounts to
-earn high yield. That's why we brought to you an initial phase of the Aave <>
-Starknet integration allowing deposit/withdrawal on Aave Ethereum by
-exclusively transacting on Starknet.
+This bridge project is the first step of Aave in Starknet ecosystem. The bridge allows users to deposit and withdraw their [aTokens](https://docs.aave.com/developers/tokens/atoken), and only `aTokens`, on StarkNet and get `static_a_tokens` instead. These `static_a_tokens` are equivalent to `aTokens`: the former grow in value when the latter grow in balance.
 
-The bridge allows users to deposit and withdraw their [aTokens](https://docs.aave.com/developers/tokens/atoken), and only `aTokens`, on StarkNet and get
-`static_a_tokens` - wrappers converting balance-increasing `aTokens` into
-exchange-rate-increasing `static_a_tokens`. We assume that L1 tokens approved by the bridge are pre-validated tokens, and that they are not deflationary.
+Holding L1 aTokens lets you earn more tokens via two different mechanisms: (i) the amount of aTokens you hold increases over time and (ii) holding aTokens allows you to claim an accruing amount of Aave reward tokens. This bridge offers both mechanisms thanks to `static_a_tokens` on L2, and the equivalent of L1 Aave reward token on L2.
 
-The bridge is also shaped for liquidity providers who are able to assume the
-Ethereum gas cost of deposits and withdrawals as they transact large enough
-amounts. They will deposit on Aave Ethereum, bridge the `static_a_tokens` to
-Starknet and make them available for users there to buy and hold, accruing this
-way yield from L1.
+The bridge is also shaped for liquidity providers who are able to assume the Ethereum gas cost of deposits and withdrawals as they transact large enough amounts. They will deposit on Aave Ethereum, bridge the `static_a_tokens` to Starknet and make them available for users there to buy and hold, thus accruing yield from L1.
+
+We assume that L1 tokens approved by the bridge are pre-validated tokens, and that they are not deflationary.
```

Comment:
shouldn't we highlight this sentence?

Reply:
I think it's ok, most people won't care about this sentence. It's just an answer to Peckshield note.